### ISSUE TITLE
Typescript types do not line up with API documentation for Tasks#addTask

### ISSUE THREAD

**Issue by schickm** [crowdin/crowdin-api-client-js#113]
### Steps to reproduce:
Given the following code, run it through the typescript compiler:

```typescript
import {
  Tasks,
} from '@crowdin/crowdin-api-client';


return new Tasks({ token: 'foo-bar-baz-blah' }).addTask(1111111, {
  title: 'translate',
  fileIds: [3],
  languageId: 'es',
  type: 0,
  assignees: [{ id: 1 }],
});
```

### Expected behavior: 

This should compile as the specified arguments handed to `addTask` are what are marked as required in the docs at https://support.crowdin.com/api/v2/#operation/api.projects.tasks.post

### Actual behavior:

I get the following typescript error due to me only supplying the `id` property of the `assignee`
```
TSError: ⨯ Unable to compile TypeScript:
error TS2739: Type '{ id: number; }' is missing the following properties from type 'Assignee': username, fullName, avatarUrl, wordsCount, wordsLeft

       assignees: [{ id: 1 }],
                   ~~~~~~~~~~~~~~~~~~~~
```

From reading the docs, it seems like I should only have to pass `id` and nothing else.  But the exported interface makes all of the properties required:

https://github.com/crowdin/crowdin-api-client-js/blob/master/src/tasks/index.ts#L217

I'd assume the correct fix for this is to just mark all but `id` as optional by tacking on a `?` at the end of them.

**Comment by schickm** [comment]
For anyone who encounters this, you can work around it my just adding a Type Assertion like so:

```typescript
import {
  Tasks,
  TasksModel
} from '@crowdin/crowdin-api-client';


return new Tasks({ token: 'foo-bar-baz-blah' }).addTask(1111111, {
  ...
  assignees: [{ id: 1 } as TasksModel.Assignee],
});
```

**Comment by yevheniyJ** [comment]
@schickm Thanks for registering an issue for this. It's fixed now in v1.11.3
